egui_demo_app/Cargo.toml

```diff
@@ -45,7 +45,7 @@ egui_demo_lib = { version = "0.18.0", path = "../egui_demo_lib", features = ["ch
 
 # Optional dependencies:
 
-bytemuck = { version = "1.9.1", optional = true }
+bytemuck = { version = "1.7.1", optional = true }
```

Comment:
why roll back to an earlier version?

Reply:
I noticed that the other crates use 1.7, so this should avoid building two different versions. But I can change it back or update all of them if you like.